lib/compiler.js

```diff
@@ -47,6 +47,10 @@ export default class Compiler {
       args.push("-bibtex");
     }
 
+    if (atom.config.get("latex-plus.synctexEnabled")) {
+      args.push("-synctex=1");
+    }
+
```

Comment:
I don't see any reason to ever disable syncTeX, so perhaps such a setting isn't necessary.